capability.go

```diff
@@ -381,6 +388,65 @@ func (c Client) SendCall(ctx context.Context, s Send) (*Answer, ReleaseFunc) {
 	return ans, rel
 }
 
+// SendStreamCall is like SendCall except that:
+//
+// 1. It does not return an answer for the eventual result.
+// 2. If the call returns an error, all future calls on this
+//    client will return the same error (without starting
+//    the method or calling PlaceArgs).
+func (c Client) SendStreamCall(ctx context.Context, s Send) error {
+	var (
+		streamError error
+		wg          *sync.WaitGroup
+	)
+	syncutil.With(&c.mu, func() {
+		streamError = c.streamError
+		if streamError == nil {
+			if c.streamWg == nil {
+				c.streamWg = &sync.WaitGroup{}
+			}
+			wg = c.streamWg
+			wg.Add(1)
+		}
+	})
+	if streamError != nil {
+		return streamError
+	}
+	ans, release := c.SendCall(ctx, s)
+	go func() {
+		defer wg.Done()
+		_, err := ans.Future().Ptr()
+		release()
+		if err != nil {
+			syncutil.With(&c.mu, func() {
+				c.streamError = err
+			})
+		}
+	}()
+	return nil
+}
+
+// WaitStreaming waits for all outstanding streaming calls (i.e. calls
+// started with SendStreamCall) to complete, and then returns an error
+// if any streaming call has failed.
+func (c Client) WaitStreaming() error {
+	var (
+		ret error
+		wg  *sync.WaitGroup
+	)
+	syncutil.With(&c.mu, func() {
+		wg = c.streamWg
+	})
+	if wg == nil {
+		return nil
+	}
+	wg.Wait()
```

Comment:
It seems like a lot of this logic could be abstracted away with `errgroup.Group`.  I'm fairly certain the zero-value `Group` will behave correctly, here.

Also, now that I think of it, shouldn't any outstanding calls to a streaming capability immediately fail if a previous call fails?  It seems like `errgroup.WithContext` could help with this.